modules/reitit-core/src/reitit/coercion.cljc

```diff
@@ -153,10 +171,15 @@
       (impl/fast-assoc response :body (coercer request response))
       response)))
 
-(defn request-coercers [coercion parameters opts]
-  (some->> (for [[k v] parameters, :when v]
-             [k (request-coercer coercion k v opts)])
-           (filter second) (seq) (into {})))
+(defn request-coercers
+  ([coercion parameters opts]
```

Comment:
is this old arity still used from some other ns? do we want to keep it around since it ignores the `:request` key?

Reply:
this is used in reitit frontend.

Reply:
right, and frontend doesn't really have request coercions, got it